.github/workflows/benchmark.yml

```diff
@@ -47,6 +47,7 @@ jobs:
       SPARK_BENCHMARK_NUM_SPLITS: ${{ github.event.inputs.num-splits }}
       SPARK_BENCHMARK_CUR_SPLIT: ${{ matrix.split }}
       SPARK_GENERATE_BENCHMARK_FILES: 1
+      SPARK_LOCAL_IP: 127.0.0.1
```

Comment:
@wangyum, I think you should set `HIVE_SERVER2_THRIFT_BIND_HOST` too. BTW, I think this is virtually because `InetAddress.getLocalHost`, via DNS, returns a wrong address in specific GitHub Actions machines.

Reply:
Here's the example

```
fv-az213-557.internal.cloudapp.net, 10.1.0.4, 10.1.0.4
```

Reply:
It seems `locaohost` much better. We do not need set `HIVE_SERVER2_THRIFT_BIND_HOST`.